classifiers/llm/bert_toxicity_classifier/code_snippet_refinery.md

```diff
@@ -0,0 +1,22 @@
+```python
+import requests
+import json
+
+API_KEY: str = "<API_KEY_GOES_HERE>"
+ATTRIBUTE: str = "headline" # only text attributes
+
+def bert_toxicity_classifier(record):
+    """
+    Uses toxic-bert via Hugging Face Inference API to classify toxicity in text.
+    """
+    api_token = API_KEY
```

Comment:
could not test as my HF key dsnt work for some reason

also, this makes it a premium I think?

Reply:
True!